post_info.php

```diff
@@ -0,0 +1,695 @@
+<?php
+require 'post_query_generators.php';
+require 'db/config.php';
+// header("Content-type: application/json");
+// below is the block for receiving POST request from the frontend
+try {
+    $GLOBALS["conn"] = new PDO(
+        "mysql:host=$servername;dbname=$database",
```

Comment:
Why don't you follow the same setup as the get_info.php file?

Reply:
because it is different logic, you set up one connection for the whole file. In this way, you can get the LAST_INSERT_ID() @richardzeng1